cmake/CMakeLists.txt

```diff
@@ -590,6 +590,9 @@ function(onnxruntime_set_compile_flags target_name)
     endif()
     if (onnxruntime_ENABLE_CPU_FP16_OPS)
       target_compile_definitions(${target_name} PRIVATE ENABLE_CPU_FP16_TRAINING_OPS)
+    endif()
+	if(onnxruntime_DISABLE_ABSEIL)
+	  target_compile_definitions(${target_name} PRIVATE DISABLE_ABSEIL)
```

Comment:
why do we need to define DISABLE_ABSEIL both here and in cmake/external/onnxruntime_external_deps.cmake?